client/components/Related-items-and-comparisons/Card.jsx

```diff
@@ -8,34 +8,90 @@ import Comparison from './Comparison.jsx';
 
 const Card = (props) => {
 
-//if(props.title === "RELATED PRODUCTS")
+  const [cardProduct, setCardProduct] = useState(null);
+// for below, maybe promise.all() to fire off all the axios.get reqs at the same time to enhance the loading speed
 
-// Product category(req.body.category):
-// Product name (req.body.name):
-// Product price (req.body.default_price):
+  useEffect(() => {
+    let cardObject = {};
+    axios.get(`/products/${props.relatedProductID}`)
+      .then((res) => {
+        cardObject.name = res.data.name;
+        cardObject.default_price = res.data.default_price;
+        return axios.get(`/products/${props.relatedProductID}/styles`) // will not work until Anisah merges changes containing styles axios.get in server
```

Comment:
Confirm that this does work, now that Anisah's styles axios.get has been merged

Reply:
Good to go!